schutzbot/mockbuild.sh

```diff
@@ -109,6 +109,9 @@ make srpm
 # override template repositories
 template_override
 
+greenprint "🎟 Adding user to mock group"
+sudo usermod -a -G mock "$(whoami)"
+
 # Compile RPMs in a mock chroot
 greenprint "🎁 Building RPMs with mock"
 sudo mock -r $MOCK_CONFIG --no-bootstrap-chroot \
```

Comment:
I believe you also want to remove the `sudo` here.

Reply:
Totally, woops :penguin: